backend/src/main/java/ca/bc/gov/backendstartapi/entity/seedlot/Seedlot.java

```diff
@@ -0,0 +1,203 @@
+
+  @Column(name = "bc_source_ind")
+  private Boolean sourceInBc;
+  // endregion
+
+  // region Collection
+  @Column(name = "collection_client_number", length = 8)
+  private String collectionClientNumber;
+
+  @Column(name = "collection_locn_code", length = 2)
+  private String collectionLocationCode;
+
+  @Column(name = "collection_start_date")
+  private LocalDateTime collectionStartDate;
+
+  @Column(name = "collection_end_date")
+  private LocalDateTime collectionEndDate;
+
+  @Column(name = "no_of_containers", precision = 6, scale = 2)
+  private BigDecimal numberOfContainers;
+
+  @Column(name = "vol_per_container", precision = 6, scale = 2)
+  private BigDecimal containerVolume;
+
+  /**
+   * Usually, but not necessarily, the product of {@link #numberOfContainers} and {@link
+   * #containerVolume}.
+   */
+  @Column(name = "clctn_volume", precision = 6, scale = 2)
+  private BigDecimal totalConeVolume;
+  // endregion
+
+  // region Interim storage
+  @Column(name = "interm_strg_client_number", length = 8)
+  private String interimStorageClientNumber;
+
+  @Column(name = "interm_strg_locn_code", length = 2)
+  private String interimStorageLocationCode;
+
+  @Column(name = "interm_strg_st_date")
+  private LocalDateTime interimStorageStartDate;
+
+  @Column(name = "interm_strg_end_date")
+  private LocalDateTime interimStorageEndDate;
+
+  @Column(name = "interm_facility_code", length = 3)
+  private String interimStorageFacilityCode;
+  // endregion
+
+  // region Orchard
+  @Column(name = "female_gametic_mthd_code", length = 4)
+  private String femaleGameticContributionMethod;
+
+  @Column(name = "male_gametic_mthd_code", length = 4)
+  private String maleGameticContributionMethod;
+
+  @Column(name = "controlled_cross_ind")
+  private Boolean producedThroughControlledCross;
+
+  @Column(name = "biotech_processes_ind")
+  private Boolean producedWithBiotechnologicalProcesses;
+
+  @Column(name = "pollen_contamination_ind")
+  private Boolean pollenContaminationPresentInOrchard;
+
+  @Column(name = "pollen_contamination_pct")
+  private Integer pollenContaminationPercentage;
+
+  @Column(name = "contaminant_pollen_bv", precision = 4, scale = 1)
+  private BigDecimal pollenContaminantBreedingValue;
+
+  @Column(name = "pollen_contamination_mthd_code", length = 4)
+  private String pollenContaminationMethodCode;
+  // endregion
+
+  // region Parent tree & SMP
+  @Column(name = "total_parent_trees")
+  private Integer totalParentTrees;
+
+  // SMP: Supplemental mass pollination.
+  @Column(name = "smp_success_pct")
+  private Integer smpSuccessPercentage;
+
+  @Column(name = "effective_pop_size", precision = 5, scale = 1)
+  private BigDecimal effectivePopulationSize;
+
+  @Column(name = "tested_parent_tree_cont_pct", precision = 6, scale = 2)
+  private BigDecimal testedParentTreeContributionPercentage;
+
+  @Column(name = "coancestry", precision = 20, scale = 10)
+  private BigDecimal coancestry;
```

Comment:
There doesn't seem to be a standard form: my browser's dictionary says that *co-ancestry* is the right form, but I could find uses for both *coancestry* and *co-ancestry*. So I opted to go with the non-hyphenated form because it's used in the description of the column in the feature data mapping spreadsheet.